src/Command/Generate/PluginConditionCommand.php

```diff
@@ -97,9 +97,9 @@ protected function interact(InputInterface $input, OutputInterface $output)
     {
         $io = new DrupalStyle($input, $output);
 
-        $entity_manager = $this->getEntityManager();
+        $entityTypeRepository = $this->getService('entity_type.repository');
```

Comment:
@naveenvalecha  I like this approach better, no public functions to maintain and keep once @stable is tagged.